core/trino-main/src/test/java/io/trino/sql/query/TestListagg.java

```diff
@@ -401,7 +393,34 @@ SELECT listagg(value, ',') WITHIN GROUP (ORDER BY id) FILTER (WHERE id % 2 = 0)
                 FROM (
                      VALUES (1, 'a'), (2, 'b'), (3, 'c'), (4, 'd')
                 ) t(id, value)
-                 """))
+                """))
                 .matches("VALUES VARCHAR 'b,d'");
     }
+
+    @Test
+    void testListaggWindow()
+    {
+        assertThat(assertions.query(
+                """
+                SELECT a, listagg(DISTINCT c) WITHIN GROUP (ORDER BY c DESC) OVER w
```

Comment:
@losipiuk could you please 🙏  add also an example in the docs https://trino.io/docs/current/functions/aggregate.html#listagg ?